test/steps/init.go

```diff
@@ -21,6 +21,18 @@ func (s *StepContext) StartStudioAPI(listen string) error {
 		_, _ = io.WriteString(w, "")
 	})
 	router.POST("/api/access-control/gate", func(w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
+		body, err := io.ReadAll(req.Body)
+		if err != nil {
+			panic(err)
+		}
+		gateRequest := make(map[string]interface{})
+		err = json.Unmarshal(body, &gateRequest)
+		if err != nil {
+			panic(err)
```

Comment:
Would it work to return an HTTP error response instead? No worries if not, just thinking about the readability of the logs when this happens

Reply:
Added a specific step to check 👍